tests/unittests/Foundation/NSIndexSetTests.mm

```diff
@@ -27,18 +27,20 @@ void verifyIndexSetEqualsArray(NSArray* expected, NSIndexSet* actual) {
     }];
 
     ASSERT_EQ([expected count], callCount);
-    ASSERT_EQ(YES, [actual isEqualToIndexSet:expectedIndexSet]);
+    ASSERT_TRUE([actual isEqualToIndexSet:expectedIndexSet]);
```

Comment:
Good point. We have not overridden isEqual: method 

---
In reply to: [73635392](https://github.com/Microsoft/WinObjC/pull/796#discussion_r73635392) [](ancestors = 73635392)